dotCMS/src/main/java/com/dotmarketing/portlets/categories/business/CategoryFactoryImpl.java

```diff
@@ -601,16 +638,11 @@ protected void deleteChildren(String inode) {
 			try {
 				conn.rollback();
 			} catch (SQLException e1) {
-				e1.printStackTrace();
+				Logger.error(CategoryFactoryImpl.class,e1);

```

Comment:
I think you can keep just one logger inside a finally clause

Reply:
true. That's what I had in mind but never did